CMakeLists.txt

```diff
@@ -38,4 +38,11 @@ add_executable(traktor_kontrol_s4_mk1_driver_linux
         classes/UtilsHelper.cpp
         main.cpp)
 
-target_link_libraries(traktor_kontrol_s4_mk1_driver_linux rtmidi libevdev.so.2 libasound.so.2 libpthread.so.0)
+find_package(PkgConfig REQUIRED)
```

Comment:
Maybe we need to think about adding sections and comments to this file. But it is enough for the moment. Thanks a lot!